client/src/pages/Search/index.js

```diff
@@ -14,7 +14,10 @@ import useMenuModal from "../../hooks/useMenuModal";
 export default function Search() {
   const dispatch = useDispatch();
   const [restaurants, setRestaurants] = useState([]);
-  const [searchFields, setSearchFields] = useState(DEFAULT_SEARCH_QUERY);
+  const [searchFields, setSearchFields] = useState({
+    ...DEFAULT_SEARCH_QUERY,
+    ...JSON.parse(localStorage.getItem("budget-eats-cache")),
+  });
   const [viewMode, setViewMode] = useState("map");
```

Comment:
can we also retain the previous view?

Reply:
what do you mean by that?

Reply:
like if they were on grid or map view, you want us to retain the view they were on?

Reply:
yeah exactly. Sorry for the late response
Consider if the user prefers grid view.
I go to the restaurant page
I go back - its map view then i gotta go to grid view
rinse repeat, kinda annoying